mappings/InventoryID-LanguageCode-Glottocode-LanguageName-Source.tsv

```diff
@@ -0,0 +1,3017 @@
+519	pon	pohn1238	POHNPEIAN	upsid
+520	prk	para1301	PARAUK	upsid
+521	pst	cent1973	PASHTO	upsid
+522	pwn	paiw1248	PAIWAN	upsid
+523	pww	pwon1235	PHLONG	upsid
+524	quh	sout2991	QUECHUA	upsid
+525	qui	quil1240	QUILEUTE	upsid
+526	rgr	resi1247	RESIGARO	upsid
+527	ron	roma1327	ROMANIAN	upsid
+528	roo	roto1249	ROTOKAS	upsid
+529	rro	waim1251	RORO	upsid
+530	rus	russ1263	RUSSIAN	upsid
+531	rut	rutu1240	RUTUL	upsid
+532	sad	sand1273	SANDAWE	upsid
+533	sag	sang1328	SANGO	upsid
+534	sah	yaku1245	YAKUT	upsid
+535	sed	seda1262	SEDANG	upsid
+536	see	sene1264	SENECA	upsid
+537	sef	ceba1235	SENADI	upsid
+538	sel	selk1253	SELKUP	upsid
+539	ses	koyr1242	SONGHAI	upsid
+540	set	nucl1632	SENTANI	upsid
+541	shb	nina1238	SHIRIANA	upsid
+542	shi	tach1250	SHILHA	upsid
+543	shs	shus1248	SHUSWAP	upsid
+544	sht	shas1239	SHASTA	upsid
+545	sin	sinh1246	SINHALESE	upsid
+546	sja	epen1239	EPENA PEDEE	upsid
+547	skd	sout2985	SIERRA MIWOK	upsid
+548	slc	sali1298	SALIBA	upsid
+549	sma	sout2674	SAAMI	upsid
+550	snn	sion1247	SIONA	upsid
+551	snv	saba1265	SA'BAN	upsid
+552	som	soma1255	SOMALI	upsid
+553	spa	stan1288	SPANISH	upsid
+554	spl	sele1250	SELEPET	upsid
+555	sqt	soqo1240	SOCOTRI	upsid
+556	srq	siri1273	SIRIONO	upsid
+557	ntu	natu1246	NAMBAKAENGO	upsid
+558	sue	suen1241	SUENA	upsid
+559	svs	savo1255	SAVOSAVO	upsid
+560	swi	suii1243	SUI	upsid
+561	taj	east2347	TAMANG	upsid
+562	tay	atay1247	ATAYAL	upsid
+563	tba	aika1237	HUARI	upsid
+564	tbi	gaam1241	TABI	upsid
+565	tca	ticu1245	TICUNA	upsid
+566	tcy	tulu1258	TULU	upsid
+567	teh	tehu1242	TEHUELCHE	upsid
+568	tel	telu1262	TELUGU	upsid
+569	tem	timn1235	TEMNE	upsid
+570	teq	nucl1339	TEMEIN	upsid
+571	tet	tetu1245	TETUN	upsid
+572	tgc	tiga1245	TIGAK	upsid
+573	tgl	taga1270	TAGALOG	upsid
+574	tha	thai1261	THAI	upsid
+575	thv	taha1241	TAMASHEQ	upsid
+576	tig	tigr1270	TIGRE	upsid
+577	tiw	tiwi1244	TIWI	upsid
+578	tiy	tiru1241	TIRURAY	upsid
+579	tli	tlin1245	TLINGIT	upsid
+580	tlo	talo1250	JOMANG	upsid
+581	tma	tama1331	TAMA	upsid
+582	tml	tamn1235	ASMAT	upsid
+583	tna	taca1256	TACANA	upsid
+584	tnl	lena1238	LENAKEL	upsid
+585	top	papa1238	TOTONAC	upsid
+586	tpm	tamp1252	TAMPULMA	upsid
+587	tpx	acat1239	TLAPANEC	upsid
+588	tpy	trum1247	TRUMAI	upsid
+589	tqo	toar1246	TAORIPI	upsid
+590	tqw	tonk1249	TONKAWA	upsid
+591	trg	lish1246	NEO-ARAMAIC	upsid
+592	tsi	nucl1649	TSIMSHIAN	upsid
+593	tsu	tsou1248	TSOU	upsid
+594	tsz	pure1242	TARASCAN	upsid
+595	ttr	tera1251	TERA	upsid
+596	tun	tuni1252	TUNICA	upsid
+597	tur	nucl1301	TURKISH	upsid
+598	twf	nort1550	PICURIS	upsid
+599	tyv	tuvi1240	TUVA	upsid
+600	tzh	tzel1254	TZELTAL	upsid
+601	ung	ngar1284	NGARINJIN	upsid
+602	unr	mund1320	MUNDARI	upsid
+603	uzn	nort2690	UZBEK	upsid
+604	vam	vani1248	VANIMO	upsid
+605	vie	viet1252	VIETNAMESE	upsid
+606	vmb	mbab1239	MBABARAM	upsid
+607	wao	wapp1239	WAPPO	upsid
+608	wap	wapi1253	WAPISHANA	upsid
+609	wba	wara1303	WARAO	upsid
+610	wgi	nucl1620	WAHGI	upsid
+611	wic	wich1260	WICHITA	upsid
+612	wim	wikm1247	WIK-MUNKAN	upsid
+613	wnw	NA	WINTU	upsid
```

Comment:
just so I'm clear: the `NA` here is because we don't have enough info in UPSID to know whether this was [Nuclear Wintu](http://glottolog.org/resource/languoid/id/nucl1651) or [Nomlaki](http://glottolog.org/resource/languoid/id/noml1242)?  If that's the case, how do we know that the right ISO code is `wnw` and not `nol`?  cf note on http://glottolog.org/resource/languoid/id/wint1259